Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -1125,13 +1131,22 @@ def extract_from_xml(xml, path):
 
 def generate_field_contents(client, fields_values, level_fields, depth):
     if fields_values and not isinstance(fields_values, dict):
-        demisto.debug(f"fields values are: {fields_values}")
-        fields_values = re.sub(r'\\(?!")', r"\\\\", fields_values)
-        demisto.debug(f"fields values after escaping: {fields_values}")
+        demisto.debug(f"Fields values string before escaping: {fields_values}")
+        # Escape backslashes if not followed by:
+        # \" - escaped double quote        \b - backspace
+        # \f - form feed                   \n - new line
+        # \r - carriage return             \t - tab
+        # \u - unicode character
+        fields_values = re.sub(r'\\(?!["bfnrtu])', r"\\\\", fields_values)
+        demisto.debug(f"Fields values string after escaping: {fields_values}")
+
         try:
+            demisto.debug(f"Loading JSON fields values string: {fields_values}")
             fields_values = json.loads(fields_values)
-        except Exception:
-            raise Exception("Failed to parse fields-values argument")
+        except Exception as e:
+            raise Exception(f"Failed to load JSON fields values string: {fields_values}. Error: {str(e)}")
+        else:
+            demisto.debug(f"Successfully loaded JSON fields values string: {fields_values}")
```

Comment:
Lets avoid putting again the fields_value as it will show in the line before in the logs.

Reply:
Removed